messages/src/main/java/life/genny/messages/managers/QEmailMessageManager.java

```diff
@@ -42,7 +42,7 @@ public void sendMessage(BaseEntity templateBe, Map<String, Object> contextMap) {
 		BaseEntity recipientBe = (BaseEntity) contextMap.get("RECIPIENT");
 		BaseEntity projectBe = (BaseEntity) contextMap.get("PROJECT");
 
-		recipientBe = beUtils.getBaseEntity(recipientBe.getCode());
+		recipientBe = beUtils.getBaseEntity(recipientBe.getCode(),true);
```

Comment:
Avoid bundling the attributes. Instead, use EntityAttributeUtils to fetch individual values.